hrpsys_ros_bridge_tutorials/src/hrpsys_ros_bridge_tutorials/urata_hrpsys_config.py

```diff
@@ -271,6 +277,73 @@ def setStAbcParametersURATALEG (self):
     def setStAbcParametersYSTLEG (self):
         print "Not implemented yet"
 
+    def setStAbcParametersCHIDORI (self):
+        # abc setting
+        abcp=self.abc_svc.getAutoBalancerParam()[1]
+        #abcp.default_zmp_offsets=[[0.015, 0.0, 0.0], [0.015, 0.0, 0.0]];
+        abcp.default_zmp_offsets=[[0.0, 0.0, 0.0], [0.0, 0.0, 0.0]];
+        abcp.move_base_gain=0.8
+        self.abc_svc.setAutoBalancerParam(abcp)
+        # kf setting
+        kfp=self.kf_svc.getKalmanFilterParam()[1]
+        kfp.R_angle=1000
+        self.kf_svc.setKalmanFilterParam(kfp)
+        # st setting
+        stp=self.st_svc.getParameter()
+        stp.st_algorithm=OpenHRP.StabilizerService.EEFM
+        stp.k_brot_p=[0, 0]
+        stp.k_brot_tc=[1000, 1000]
+        stp.eefm_body_attitude_control_gain=[0.5, 0.5]
+        stp.eefm_body_attitude_control_time_const=[1000, 1000]
+        stp.eefm_rot_damping_gain=20*1.6*1.1*1.5
```

Comment:
Please include 
https://github.com/start-jsk/rtmros_tutorials/pull/401
commit.
Update required for `eefm_rot_damping_gain`, `eefm_pos_damping_gain`, `eefm_rot_time_const`, `eefm_pos_time_const_support`, and `eefm_leg_*_margin`.